### ISSUE TITLE
Infinite Membership duration

### ISSUE THREAD

**Issue by TomekFraczek** [ExcursionClub/ExCSystem#18]
Certain users (ie admin) should never have their membership expire. This is represented by setting their expiration date to be the largest date representable by the datetime field. Setting it to this value occurs by adding the max timedelta to the current expiration date. This however results in an overflow error which, as of now, is quite haphazardly handled.

**Comment by estensen** [comment]
@TomekFraczek have you thought more on how you can handle this?

**Comment by TomekFraczek** [comment]
Not really. The way it's done right now (try to add the passed max timedelta and if and overflow exception occurs, set the expiration time to it's max time) works quite well. The issue is mostly that it feels kinda clunky and like it could potentially be exploited.
